packages/@react-stately/color/src/Color.ts

```diff
@@ -70,6 +70,10 @@ export class Color {
     }
   }
 
+  toHexInt(): number {
+    return parseInt(this.toString('hex').substring(1), 16);
```

Comment:
I think we can do this a little faster, after all, hex doesn't matter, it's just a number
what if this function didn't need to parse a string and just did something like this
```
toInt(): number {
  return this.value.toRGB().toInt();
}
```

where toInt() on the RGB class is
```
return this.red * 256 + this.green * 16 + this.blue;
```

Reply:
I don't know what it'd mean for the other types, but we can start here